### PR TITLE
remove nelmio >= 3.0 conflict rule

### LINKED ISSUE #988: Be able to use Nelmio 3.0 Version
I saw Nelmio 3.0 version is in conflict since 30 June 2016 - [See the commit](https://github.com/sonata-project/SonataUserBundle/commit/fcef4a6f3ab4d8d3992fe17b7f091b1fd6d8a899#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780).
I really need to use the 3.0 version, when do you think we will be able to use it ?

Thanks,

Jérôme

### PR DESCRIPTION
**Pull request by ibogdan94**
Hi, guys 
I am targeting this branch, because i have conflict with nelmio/api-doc-bundle. SonataUserBundle has description in composer.json => section conflict. Now, I need to migrate to Symfony4 and want to use latest packages as much as possible

Related: https://github.com/sonata-project/SonataUserBundle/pull/989#issuecomment-358578412

Closes #989
Closes #988

## Changelog

```markdown
Removed
- removed conflict rule for nelmio/api-doc-bundle inside composer.json
```

## Subject
https://www.dropbox.com/s/qf14nqwc07rszqt/Selection_292.jpg?dl=0
Try to install sonata user bundle and received an error message:
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - sonata-project/user-bundle dev-master conflicts with nelmio/api-doc-bundle[v3.0.0].
    - sonata-project/user-bundle dev-master conflicts with nelmio/api-doc-bundle[v3.0.0].
    - Installation request for sonata-project/user-bundle dev-master -> satisfiable by sonata-project/user-bundle[dev-master].
    - Installation request for nelmio/api-doc-bundle ^3.0 -> satisfiable by nelmio/api-doc-bundle[v3.0.0].

Installation failed, reverting ./composer.json to its original content.

>

### CODE REVIEW

**Comment by greg0ire** [comment]
Please associate your gmail address with your Github account, or change the email in your commits to an address already associated with it.

**Comment by ibogdan94** [comment]
@greg0ire my github account already associate my gmail address

**Comment by greg0ire** [comment]
Make sure it is spelt right like on this page: https://github.com/sonata-project/SonataUserBundle/pull/998/commits/407c914cb7e199c27149aab4476ceef408ba0264 (hover the question mark to see it), there might be a typo.

**Comment by ibogdan94** [comment]
@greg0ire all looks fine. I do not understand the question. When do you plan to merge these changes?

**Comment by ibogdan94** [comment]
@greg0ire if you need my email address, you can see it on profile page
https://github.com/ibogdan94

**Comment by greg0ire** [comment]
I don't want to expose your email so I'm going to send one to you.

**Comment by greg0ire** [comment]
> When do you plan to merge these changes?

I won't merge it, because I never merge things that are not approved by someone else. I restarted the build.

File: composer.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "conflict": {
         "friendsofsymfony/rest-bundle": "<1.1",
         "jms/serializer": "<0.13 || >=2.0",
-        "nelmio/api-doc-bundle": "<2.4 || >= 3.0",
+        "nelmio/api-doc-bundle": "<2.4",
```
Comment (OskarStark) [inline composer.json]: would change it to `^2.4 || ^3.0`

Comment (ibogdan94) [reply composer.json]: No, it’s would not work

Comment (ibogdan94) [reply composer.json]: No, it’s would not work

Comment (OskarStark) [reply composer.json]: ah this is the conflict rule, sorry

Comment (ibogdan94) [reply composer.json]: Thanks ;)

**Comment by greg0ire** [comment]
Now that this is merged, your only option is to associate what I think is your old gmail address with your github account. Can you please do that?

**Comment by OskarStark** [comment]
> Now that this is merged, your only option is to associate what I think is your old gmail address with your github account. Can you please do that?

oh sorry @greg0ire, didn't recognize is, next time we should request it to prevent merge

**Comment by greg0ire** [comment]
It can be fixed afterwards, no big deal.

**Comment by ibogdan94** [comment]
@greg0ire still didn't get what do you want from me

**Comment by OskarStark** [comment]
please go to your github settings and add this mailadress:
screenshot 2018-01-18 12 09 46

**Comment by ibogdan94** [comment]
@OskarStark could you check it one more time? I added an one more email to m github account.
I didn't see this question mark 
image

**Comment by OskarStark** [comment]
thank you 👍

**Pull request merged**